pallets/oracle/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 //! # Oracle Pallet
 //! Based on the [specification](https://spec.interlay.io/spec/oracle.html).
 
-#![deny(warnings)]
+#![warn(warnings)]
```

Comment:
Is this change necessary or did you forget to change it back?

Reply:
@ebma 
I did it because the test failed because of `unused import: `DispatchResult``

Should I fix it or we can leave  it like this?

Reply:
Can't we just remove the unused import then?

Reply:
No. Because `DispatchResult` for other code when there is no feature `testing-utils`

Reply:
I fixed it by conditionally importing it.